Makefile

```diff
@@ -82,9 +82,10 @@ _dev-with-widgets:
 		--user jovyan \
 		-v `pwd`/jupyter_dashboards:$(EXTENSION_DIR) \
 		-v `pwd`/scripts/jupyter-dashboards:/usr/local/bin/jupyter-dashboards \
+		-v `pwd`/../declarativewidgets:/declarativewidgets \
 		-v `pwd`/etc/notebooks:/home/jovyan/work \
 		$(BOWER_REPO) bash -c '$(LANG_SETUP_CMD) && $(EXT_DEV_SETUP) && \
-			pip install jupyter_declarativewidgets && \
+			pip install --no-binary :all: $$(ls -1 /declarativewidgets/dist/*.tar.gz | tail -n 1) && \
```

Comment:
Actually, these weren't inadvertent. I switched us to using the latest stable decl widget release by default a while back. It's the more common case.